contents/blog/operators-and-extractors-technical-deep-dive/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: How Operators and Extractors Work Under-the-Hook
+date: 2022-08-01
+author: Benji Lampel
+image: ./image.svg
+banner: ./banner.svg
+description: A technical deep-dive on how the Airflow OSS and OpenLineage OSS projects interact. 
+---
+
+## Overview
+
+Airflow Operators and OpenLineage Extractors have a specific, if quirky, way of working together. Recently, the way they work together has seen a bit of an overhaul, and the new SQL Check Extractors added a new and unique way that the extractors work and interact with operators. In this blog, we'll demystify these relationships.
+Note: This blog post describes the relationships of the operators and extractors only for `Airflow>=2.3` and `OpenLineage>=0.12.0`.
```

Comment:
This Markdown renderer is collapsing this into a single paragraph without a blank line in between - this happens throughout.